packages/grid/x-data-grid/src/components/GridRow.tsx

```diff
@@ -329,6 +340,7 @@ function GridRow(props: React.HTMLAttributes<HTMLDivElement> & GridRowProps) {
           tabIndex={tabIndex}
           className={clsx(classNames)}
           colSpan={colSpan}
+          enableDragEvents={enableDragEvents}
```

Comment:
The prop name can be discussed since we never used "enableSomething", but we already have "disableSomething" so I don't have a strong opinion of which path to take.

Reply:
Done